src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -160,16 +167,16 @@ AutomatedRssDownloader::~AutomatedRssDownloader()
 
 void AutomatedRssDownloader::loadSettings()
 {
-    const auto *pref = Preferences::instance();
-    resize(pref->getRssGeometrySize());
-    m_ui->hsplitter->restoreState(pref->getRssHSplitterSizes());
+    if (m_storeDialogSize.get().isValid())
+        resize(m_storeDialogSize);
+    if (!m_storeHSplitterSize.get().isEmpty())
+        m_ui->hsplitter->restoreState(m_storeHSplitterSize);
 }
 
 void AutomatedRssDownloader::saveSettings()
 {
-    Preferences *const pref = Preferences::instance();
-    pref->setRssGeometrySize(size());
-    pref->setRssHSplitterSizes(m_ui->hsplitter->saveState());
+    m_storeDialogSize = size();
+    m_storeHSplitterSize = m_ui->hsplitter->saveState();
```

Comment:
Maybe we should start to serialize sllitter states manually to be portable between versions of Qt?

Reply:
I'll think about it and it will be another PR.